code/upaya/Gemfile

```diff
@@ -2,9 +2,9 @@ source 'https://rubygems.org'
 ruby '2.3.0'
 
 gem 'rails', '~> 4.2.6'
-
 gem 'attr_encrypted'
 gem 'autoprefixer-rails', '~> 5.2'
+gem 'bullet'
```

Comment:
Do you want to merge this?   Or is it just WIP?   If you want to merge then perhaps add a TODO here to move this back to :development block?  And use the same version as below maybe?